webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,30 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+/* eslint-disable max-params  */
+
+export function notify(message, type, link, sticky = type !== 'success') {
```

Comment:
Wee comment above about un-matching function signature. I tend to prefer the hash version when there are many possible parameters, that way it's clear which parameter is which (and you will be able to remove the `eslisnt-disable` above)